src/integrations/prefect-aws/prefect_aws/workers/ecs_worker.py

```diff
@@ -1357,7 +1367,9 @@ def _prepare_task_definition(
                 container["environment"].remove(item)
 
         if configuration.configure_cloudwatch_logs:
-            prefix = f"prefect-logs_{self._work_pool_name}_{flow_run.deployment_id}"
+            prefix = f"prefect-logs_{self._work_pool_name}"
+            if flow_run.deployment_id:
+                prefix = f"{prefix}_{flow_run.deployment_id}"
```

Comment:
did you want to fall back to `flow_run.flow_id` here as well?

Reply:
Yup! Let me fix that...